sweep.yaml

```diff
@@ -0,0 +1,84 @@
+program: main.py
+command:
+  - ${env}
+  - ${interpreter}
+  - -O
+  - ${program}
+  - ${args}
+metric:
+  name: avg_reward
+  goal: maximize
+method: bayes
+parameters:
+  general_model_config:
+    parameters:
+      dim:
+        values: [ 64, 128, 256 ]
+      depth:
+        values: [ 1, 2, 3, 4 ]
+      heads:
+        values: [ 1, 2, 4, 8 ]
+      mlp_dim:
+        values: [ 128, 256, 512 ]
+      dropout:
+        distribution: uniform
+        min: 0.1
+        max: 0.25
+      emb_dropout:
+        distribution: uniform
+        min: 0.1
+        max: 0.25
+  pretrain:
+    parameters:
+      early_stopping:
+        values: [ True, False ]
```

Comment:
we actually don't have early_stopping implemented, can just remove this from config